pennylane/measurements/state.py

```diff
@@ -122,23 +122,21 @@ def circuit():
         with a compatible device.
     """
     wires = Wires(wires)
-    return StateMP(wires=wires)
+    return DensityMatrixMP(wires=wires)
 
 
 class StateMP(StateMeasurement):
     """Measurement process that returns the quantum state in the computational basis.
 
-    Please refer to :func:`state` and :func:`density_matrix` for detailed documentation.
+    Please refer to :func:`state` for detailed documentation.
 
     Args:
-        wires (.Wires): The wires the measurement process applies to.
-            This can only be specified if an observable was not provided.
         id (str): custom label given to a measurement instance, can be useful for some applications
             where the instance has to be identified
     """
 
-    def __init__(self, wires: Optional[Wires] = None, id: Optional[str] = None):
-        super().__init__(wires=wires, id=id)
+    def __init__(self, *, id: Optional[str] = None):
```

Comment:
Why is the star necessary here?

Reply:
not _super_ necessary, but without it, a user can do `StateMP([0])` and they might think it's the wires, but it'll actually set the `id`. This way, they get the error right away. Anyway, it'll be removed before the release when I add wires to StateMP :p